app/allocation/views/view.njk

```diff
@@ -119,7 +119,18 @@
   {% endif %}
 
   {% for slot in allocationPeople.filledSlots %}
+  <section class="app-!-position-relative">
```

Comment:
Maybe keep consistent indentation with this change. 

I know it comes with the catch of git thinking more has changed than actually has, but I feel it's better to maintain longterm scanability and readability of the code in this instance.

Reply:
👍